internal/runtime/network.go

```diff
@@ -47,7 +49,7 @@ type networkModule struct {
 func vatNetwork(c *cli.Context, lx fx.Lifecycle, b *metrics.BandwidthCounter) (mod networkModule, err error) {
 	mod.Vat.NS = c.String("ns")
 	mod.Vat.Host, err = libp2p.New(c.Context,
-		libp2p.NoTransports,
+		libp2p.DefaultTransports,
```

Comment:
The reason why I set to defaultTransports is to allow TCP addresses if wanted

Reply:
Was this because of the QUIC kernel-buffer issue?  If so, is this still useful?

I had kind of wanted to force the use of QUIC, but if TCP is still useful, we can certainly leave it enabled.  Note that this also enables the websocket transport (which is not a problem).

Reply:
It was in order to also test with TCP connections, but if you prefer we can undo it. Although I think that users should be able to work on TCP if they prefer